### PR TITLE
Translate rendering elements

### PR DESCRIPTION
**Pull request by pepjo**

### CODE REVIEW

File: content/docs/rendering-elements.md [hunk 1]
```diff
@@ -1,75 +1,75 @@
 ---
 id: rendering-elements
-title: Rendering Elements
+title: Renderitzant elements
 permalink: docs/rendering-elements.html
 redirect_from:
-  - "docs/displaying-data.html"
+- "docs/displaying-data.html"
```
Comment (seraco) [inline content/docs/rendering-elements.md]: This line should be the same as the original source it is translated from.

File: content/docs/rendering-elements.md [hunk 2]
```diff
@@ -1,75 +1,75 @@
...
-Elements are the smallest building blocks of React apps.
+Els elements són la unitat més petita de les aplicacions de React.
 
-An element describes what you want to see on the screen:
+Un element descriu el que voleu veure a la pantalla:
 
-```js
-const element = <h1>Hello, world</h1>;
-```
+'''js
```
Comment (seraco) [inline content/docs/rendering-elements.md]:
```suggestion
```js
```

File: content/docs/rendering-elements.md [hunk 3]
```diff
@@ -1,75 +1,75 @@
...
+const element = <h1>Hola, món</h1>;
+'''
```
Comment (seraco) [inline content/docs/rendering-elements.md]: Revise these quotes

File: content/docs/rendering-elements.md [hunk 4]
```diff
@@ -1,75 +1,75 @@
...
 
-Unlike browser DOM elements, React elements are plain objects, and are cheap to create. React DOM takes care of updating the DOM to match the React elements.
+A diferència dels elements del DOM dels navegadors, els elements de React són objectes plans, i la seva creació és de baix cost. ReactDOM es cuida d'actualitzar el DOM per sincronitzar-se amb els elements de React.
```
Comment (seraco) [inline content/docs/rendering-elements.md]:
```suggestion
A diferència dels elements del DOM dels navegadors, els elements de React són objectes plans, i la seva creació és de baix cost. React DOM es cuida d'actualitzar el DOM per sincronitzar-se amb els elements de React.
```

File: content/docs/rendering-elements.md [hunk 5]
```diff
@@ -1,75 +1,75 @@
...
+> Un podria confondre els elements amb un altre concepte anomenat "components". En la [següent secció](/docs/components-and-props.html) parlarem de components. Els elements formen els components. Us recomanem aquesta secció abans de continuar.
 
-## Rendering an Element into the DOM {#rendering-an-element-into-the-dom}
+## Renderitzant un element en el DOM {#rendering-an-element-in-the-dom}
 
-Let's say there is a `<div>` somewhere in your HTML file:
+Diguem que hi ha un '<div>' en alguna part del vostre fitxer HTML:
 
-```html
+'''html
```
Comment (seraco) [inline content/docs/rendering-elements.md]: Revise quotes

File: content/docs/rendering-elements.md [hunk 6]
```diff
@@ -1,75 +1,75 @@
...
 <div id="root"></div>
-```
+'''
```
Comment (seraco) [inline content/docs/rendering-elements.md]: Revise quotes

File: content/docs/rendering-elements.md [hunk 7]
```diff
@@ -1,75 +1,75 @@
...
+En diem un node "arrel" perquè tot el que estigui dins d'ell serà gestionat per React DOM.
 
-Applications built with just React usually have a single root DOM node. If you are integrating React into an existing app, you may have as many isolated root DOM nodes as you like.
+Les aplicacions construïdes solament amb React normalment tenen un únic node arrel al DOM. Si estàs integrant React en una aplicació existent, pots tenir tants nodes arrel del DOM aïllats com vulguis.
 
-To render a React element into a root DOM node, pass both to `ReactDOM.render()`:
+Per renderitzar un element de React en un node arrel del DOM, passa'ls tots dos a 'ReactDOM.render()':
 
-`embed:rendering-elements/render-an-element.js`
+'embed:rendering-elements/render-an-element.js'
```
Comment (seraco) [inline content/docs/rendering-elements.md]: Review quotes

File: content/docs/rendering-elements.md [hunk 8]
```diff
@@ -1,75 +1,75 @@
...
 
-[](codepen://rendering-elements/render-an-element)
+[](Codepen://rendering-elements/render-an-element)
```
Comment (seraco) [inline content/docs/rendering-elements.md]:
```suggestion
**[Prova-ho a CodePen](Codepen://rendering-elements/render-an-element)**
```

File: content/docs/rendering-elements.md [hunk 9]
```diff
@@ -1,75 +1,75 @@
...
+Els elements de React són [immutables](https://es.wikipedia.org/wiki/Objeto_inmutable). Un cop crees un element, no pots canviar els seus fills o atributs. Un element és com un fotograma solitari en una pel·lícula: aquest representa la interfície d'usuari en cert punt en el temps.
 
-With our knowledge so far, the only way to update the UI is to create a new element, and pass it to `ReactDOM.render()`.
+Amb el nostre coneixement fins a aquest punt, l'única manera d'actualitzar la interfície d'usuari és creant un nou element, i passar-ho a 'ReactDOM.render()'.
 
-Consider this ticking clock example:
+Considera aquest exemple d'un rellotge en marxa:
 
-`embed:rendering-elements/update-rendered-element.js`
+'embed:rendering-elements/update-rendered-element.js'
```
Comment (seraco) [inline content/docs/rendering-elements.md]: Quotes

File: content/docs/rendering-elements.md [hunk 10]
```diff
@@ -1,75 +1,75 @@
...
 
-[](codepen://rendering-elements/update-rendered-element)
+[](Codepen://rendering-elements/update-rendered-element)
```
Comment (seraco) [inline content/docs/rendering-elements.md]: Same as in other places, it will not be translated so far.

File: content/docs/rendering-elements.md [hunk 11]
```diff
@@ -1,75 +1,75 @@
...
+## React només actualitza el que cal {#react-only-updates-whats-necessary}
 
-React DOM compares the element and its children to the previous one, and only applies the DOM updates necessary to bring the DOM to the desired state.
+React DOM compara l'element i els seus fills amb l'element anterior, i només aplica les actualitzacions del DOM que són necessàries perquè el DOM estigui en l'estat desitjat.
 
-You can verify by inspecting the [last example](codepen://rendering-elements/update-rendered-element) with the browser tools:
+Pots fer això inspeccionant l'[últim exemple](codepen://rendering-elements/update-rendered-element) amb les eines del navegador:
 
-![DOM inspector showing granular updates](../images/docs/granular-dom-updates.gif)
+![Inspector de l'DOM mostrant actualitzacions diminutes](../images/docs/granular-dg-updates.gif)
```
Comment (seraco) [inline content/docs/rendering-elements.md]:
```suggestion
![Inspector del DOM mostrant actualitzacions diminutes](../images/docs/granular-dg-updates.gif)
```

File: content/docs/rendering-elements.md (hunk 8 above)
Comment (seraco) [inline content/docs/rendering-elements.md]:
```suggestion
[](codepen://rendering-elements/render-an-element)
```

File: content/docs/rendering-elements.md (hunk 10 above)
Comment (seraco) [inline content/docs/rendering-elements.md]:
```suggestion
[](codepen://rendering-elements/update-rendered-element)
```

**Comment by pepjo** [comment]
@seraco I think this is ready for merge. Review and merge when you are ready.

**Pull request merged**